frontend/src/theme/components/Checkbox.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentStyleConfig } from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+
+export const Checkbox: ComponentStyleConfig = {
+  parts: ['others'],
+  baseStyle: ({ theme }) => ({
+    control: {
+      borderRadius: '0.19rem',
+      border: '2px solid',
+      borderColor: 'primary.500',
+      _checked: {
+        bg: 'primary.500',
+        borderColor: 'primary.500',
+        _hover: {
+          bg: 'primary.500',
+          borderColor: 'primary.500',
+        },
+      },
+      _focus: {
```

Comment:
`_disabled` styling seems to be missing. Figma seems to not have the _disabled:_check state, so maybe ask @pearlyong  about it

Reply:
have added styles for this but I need a little help with styling cursor for disabled states for radio containers! im trying to set the cursor to 'default' when disabled but the styles aren't applying

Reply:
the cursor should be `not-allowed`

Reply:
disabled + check state seems to be wrong too, should add stories for all of the combinations.